### ISSUE TITLE
Benchmark code for cluster evaluation

### ISSUE THREAD

**Issue by YuMJie** [ServerlessLLM/ServerlessLLM#81]
Thanks for your great jobs, I am interested in the benchmark code for cluster evaluation. Could you provide the code?

**Comment by future-xy** [comment]
Yes, we will provide detailed instructions for benchmarking ServerlessLLM in a GPU cluster, similar to the [current instructions](https://github.com/ServerlessLLM/ServerlessLLM/tree/main/benchmarks) available for model loading.
